x-pack/test/functional/apps/ml/short_tests/model_management/model_list.ts

```diff
@@ -77,17 +96,32 @@ export default function ({ getService }: FtrProviderContext) {
 
       // Make sure the .ml-stats index is created in advance, see https://github.com/elastic/elasticsearch/issues/65846
       await ml.api.assureMlStatsIndexExists();
+
+      // Create ingest pipeline and destination index that's tied to model
+      await ml.api.createIngestPipeline(modelWithPipelineAndDestIndex.modelId);
+      await ml.api.createIndex(modelWithPipelineAndDestIndexExpectedValues.index, undefined, {
+        index: { default_pipeline: `pipeline_${modelWithPipelineAndDestIndex.modelId}` },
+      });
     });
 
     after(async () => {
       await ml.api.stopAllTrainedModelDeploymentsES();
       await ml.api.deleteAllTrainedModelsES();
+
+      await ml.api.cleanMlIndices();
+      await ml.api.deleteIndices(modelWithPipelineAndDestIndexExpectedValues.index);
+
       await ml.api.deleteIngestPipeline(modelWithoutPipelineDataExpectedValues.name, false);
       await ml.api.deleteIngestPipeline(
         modelWithoutPipelineDataExpectedValues.duplicateName,
         false
       );
-      await ml.api.cleanMlIndices();
+
+      // // Need to delete index before ingest pipeline, else it will give error
```

Comment:
Nit - extra set of `//`

Reply:
Updated here [`f2237b6` (#167911)](https://github.com/elastic/kibana/pull/167911/commits/f2237b6ec94313e3f2123421911fad8a8c87744a)